frontend/src/routes/Infrastructure/Clusters/DiscoveredClusters/DiscoveryConfig/discoveryConfigFilters.ts

```diff
@@ -0,0 +1,313 @@
+ * that are properly localized using i18n.
+ *
+ * Supported acronyms include:
+ * - MOA: Maps to ROSA (Red Hat OpenShift on AWS)
+ * - MOA-HostedControlPlane: Maps to ROSA with hosted control plane
+ * - ROSA: Red Hat OpenShift on AWS
+ * - ROSA-HCP: ROSA with HyperShift
+ * - OCP-ASSISTEDINSTALL: OpenShift Assisted Installer
+ * - OCP: OpenShift Container Platform
+ * - OSD: OpenShift Dedicated
+ * - OSD-TRIAL: Trial version of OpenShift Dedicated
+ * - ARO: Azure Red Hat OpenShift
+ * - RHMI: Red Hat Managed Integration
+ * - RHOIC: Red Hat OpenShift on IBM Cloud
+ *
+ * @param acronym - The cluster type acronym to convert (case insensitive)
+ * @returns The localized full display name of the cluster type, or the original acronym if not recognized
+ */
+
+export function getFullTypeByAcronymForDiscoveryClustersType(acronym: string) {
+  switch (acronym.toUpperCase()) {
+    case 'MOA':
+      return t('type.rosa')
+    case 'MOA-HOSTEDCONTROLPLANE': //MOA-HostedControlPlane
+      return t('type.rosa.hcp')
+    case 'ROSA': //ROSA (Red Hat OpenShift on AWS)
+      return t('type.rosa')
+    case 'ROSA-HYPERSHIFT': //ROSA-HyperShift (ROSA with HyperShift)
+      return t('type.rosa.hcp')
+    case 'OCP-ASSISTEDINSTALL': //OCP-AssistedInstall (OpenShift Assisted Installer)
+      return t('type.ocp.assisted.install')
+    case 'OCP': //OCP (OpenShift Container Platform)
+      return t('type.ocp')
+    case 'OSD': //OSD (OpenShift Dedicated)
+      return t('type.osd')
+    case 'OSDTRIAL': //OSDTrial (Trial version of OpenShift Dedicated)
+      return t('type.osd.trial')
+    case 'ARO': //ARO (Azure Red Hat OpenShift)
+      return t('type.aro')
+    case 'RHMI': //RHMI (Red Hat Managed Integration)
+      return t('type.rhmi')
+    case 'RHOIC': //RHOIC (Red Hat OpenShift on IBM Cloud)
+      return t('type.rhoic')
+    default:
+      // Unable to find match, return existing acronym
+      return acronym
+  }
+}
+
+// ClusterTypes is the list of cluster types to discover. These types represent the platform the cluster is running on
+
+export const DISCOVERY_CLUSTER_TYPES = [
+  'OSD',
+  'OSDTrial',
+  'OSD-Trial',
+  'OCP',
+  'RHMI',
+  'ROSA',
+  'RHOIC',
+  'MOA',
+  'MOA-HostedControlPlane',
+  'ROSA-HyperShift',
+  'ARO',
+  'OCP-ASSISTEDINSTALL',
+]
+
+// InfrastructureProviders is the list of infrastructure providers to discover. This can be a list of cloud providers or platforms (e.g., AWS, Azure, GCP) where clusters might be running.
+export const INFRASTRUCTURE_PROVIDERS = [
+  'aws',
+  'azure',
+  'baremetal',
+  'external',
+  'gcp',
+  'ibmcloud',
+  'kubevirt',
+  'libvirt',
+  'none',
+  'nutanix',
+  'openstack',
+  'ovirt',
+  'powervs',
+  'vsphere',
+]
+
+export const CLUSTER_TYPE_GROUPS: {
+  [key: string]: { displayName: string; types: string[] }
+} = {
+  OCP: {
+    displayName: t('type.group.ocp'),
+    types: ['OCP', 'OCP-ASSISTEDINSTALL'],
```

Comment:
@vishsanghishetty This needs to be stored in the YAML with the original casing, `OCP-AssistedInstall`. (It's OK that we treat things as case insensitive in our code, but they are not really on the OCM side.)

In fact, I updated the YAML for your `DiscoveryConfig` on the dev cluster to match just `OCP-ASSISTEDINSTALL` and I found zero discovered clusters showed up. But when I updated it to `OCP-AssistedInstall`, I see there are several in your OCM account:
![image](https://github.com/user-attachments/assets/7277aba7-5f08-45be-96f0-5f7642f1643e)